### PR TITLE
[WIP] Automate version number

### LINKED ISSUE #215: Explore using relic
[relic](https://github.com/spacetelescope/relic) is a version management package built and maintained by folks here at ST.  It could help us avoid having to hard code version numbers at various places within our package.

### PR DESCRIPTION
**Pull request by bourque**
This PR adds a `version.py` module and `get_version_number` function that returns the current version number of the `jwql` package.  As such, `setup.py` and `docs.source.conf.py` are modified to  make use of this new function.

Closes #215

### CODE REVIEW

**Comment by bourque** [comment]
@bhilbert4 @Johannes-Sahlmann Could one of you review this and let me know what you think?

File: setup.py [hunk 1]
```diff
@@ -2,6 +2,8 @@
 from setuptools import setup
 from setuptools import find_packages
 
+from version import get_version
+
 VERSION = '0.16.0'
```
Comment (Johannes-Sahlmann) [inline setup.py]: I think this line can be removed.

**Comment by bourque** [comment]
@Johannes-Sahlmann Excellent point.  In the case that you describe, the wrong version number would be used in the installation.  

I will have to re-think this.

**Comment by bourque** [comment]
I am going to close this, because the issue that @Johannes-Sahlmann points out would be a major issue.  For now, we will revert to using hard coded version numbers, and in the meantime continue to explore using `relic` (#215).

**Pull request closed without merging**